### ISSUE TITLE
Evaluation on Test Set isn't working 

### ISSUE THREAD

**Issue by gaurav-g-12** [google-research-datasets/hiertext#22]
I’m having trouble uploading my test set JSON file on https://rrc.cvc.uab.es/?ch=18 for evaluation, as it returns an error during result calculation. I even tried using the sample output file provided on the website, but that also failed to upload. Could you suggest a solution to resolve this issue? Thank you!

**Comment by longshangbang** [comment]
Is it the same issue as https://github.com/google-research-datasets/hiertext/issues/21?

**Comment by gaurav-g-12** [comment]
> Is it the same issue as #21?

yes, initially for 2 minutes it shows "The method is being calculated", after which it giving "The method has produced an error when calculating: Default - Paragraph PQ, Default - Line PQ"

**Comment by gaurav-g-12** [comment]
<img width="1231" alt="image" src="https://github.com/user-attachments/assets/e3ebf9b9-c7bc-4e6d-94a7-cf59884cae97">

**Comment by ShashankKrishnaV** [comment]
This is the same issue I am also facing from last few days.

**Comment by gaurav-g-12** [comment]
When this is expected to be fixed?

**Comment by mavi-iitd** [comment]
Yeah, pls fix the issue as it is really urgent to put it in my CVPR paper.

**Comment by longshangbang** [comment]
test set released: https://github.com/google-research-datasets/hiertext/commit/70b6620b2b112597d8219e11eee9773a1403827c

users can evaluate locally
